### PR TITLE
feat: add cluster update tutorial DOC-920

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR  makes the following changes: 
- Updates cluster pack versions
- Adds support for Azure regions without AZs
- Create the cluster update tutorial
- Adds `podman` to the tutorial container

## Review Changes

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-920)

### CODE REVIEW

**Review by karl-cardenas-coding — COMMENTED** [review]
Some minor suggestions and comments. Overall, this looks great.

File: Dockerfile [hunk 1]
```diff
@@ -59,6 +59,10 @@ RUN  wget https://software.spectrocloud.com/spectro-registry/cli/v$PALETTE_CLI_V
         git clone https://github.com/spectrocloud/CanvOS.git && \
         rm -rf /var/cache/apk/* 
 
+RUN tf_latest=`curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest |  grep tag_name | cut -d: -f2 | tr -d \"\,\v | awk '{$1=$1};1'` && \
```
[the remark this answers is not in the document]
Comment (addetz) [reply Dockerfile]: Understood! You're right. I didn't think about the larger size or the beta releases. I have added an `ARG`, consolidated the `RUN` block and removed the zip file after install too.

File: terraform/iaas-cluster-deployment-tf/inputs.tf [hunk 2]
```diff
@@ -48,6 +48,11 @@ variable "azure_resource_group" {
   default     = ""
 }
 
+variable "azure-use-azs" {
+  type        = bool
+  description = "A flag for configuring whether to use Azure Availability Zones. Check if your Azure region provides them."
```
[the remark this answers is not in the document]
Comment (addetz) [reply terraform/iaas-cluster-deployment-tf/inputs.tf]: Thanks for taking the time to do this! 🙇‍♀️  I have updated other descriptions for this input as well.

File: terraform/iaas-cluster-update-tf/inputs.tf [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+}
+
+variable "azure-use-azs" {
+  type        = bool
+  description = "A flag for configuring whether to use Azure Availability Zones. Check if your Azure region provides them."
+}
+
+variable "aws-region" {
+  type        = string
+  description = "AWS region"
```
[the remark this answers is not in the document]
Comment (addetz) [reply terraform/iaas-cluster-update-tf/inputs.tf:58]: I've added `us-east-1` as default. Let me know if you prefer another

File: docs/docker.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
 To download the image issue the following commnad. Change the version tag as needed.
 
 ```shell
-docker pull ghcr.io/spectrocloud/tutorials:1.0.2
+docker pull ghcr.io/spectrocloud/tutorials:1.0.12
```
Comment (addetz) [inline docs/docker.md:6]: Bumping to current version+1 as this will be a new release

**Comment by karl-cardenas-coding** [comment]
@addetz could please also update the README with a markdown link to the new tutorial code 😄

**Pull request merged**